android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -221,6 +231,9 @@ public void handleMessage(Message msg) {
             switch (msg.what) {
                 case SHOW_PROGRESS:
                     if (player != null) {
+                        if (isPlayingAd()) {
```

Comment:
I think you should check that controls are enable or at least playerControlView is not null